evap/contributor/tests/test_forms.py

```diff
@@ -63,6 +63,145 @@ def test_staff_only(self):
         self.assertEqual(expected, set(formset.forms[0].fields['questionnaires'].queryset.all()))
         self.assertEqual(expected, set(formset.forms[1].fields['questionnaires'].queryset.all()))
 
+    def test_editors_cannot_degrade_responsibles(self):
+        course = mommy.make(Course)
+        user = mommy.make(UserProfile)
+        contribution = mommy.make(Contribution, course=course, contributor=user, responsible=True,
+                                  can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+        InlineContributionFormset = inlineformset_factory(Course, Contribution, formset=ContributionFormSet, form=EditorContributionForm, extra=1)
+
+        data = {
+            "contributions-TOTAL_FORMS": "1",
+            "contributions-INITIAL_FORMS": "1",
+            "contributions-MIN_NUM_FORMS": "0",
+            "contributions-MAX_NUM_FORMS": "1000",
+            "contributions-0-course": "{}".format(course.pk),
+            "contributions-0-order": "1",
+            "contributions-0-id": "{}".format(contribution.pk),
+            "contributions-0-contributor": "{}".format(user.pk),
+            "contributions-0-does_not_contribute": "on",
+            "contributions-0-responsibility": "RESPONSIBLE",
+            "contributions-0-comment_visibility": "OWN",
+            "contributions-0-label": "",
+            "contributions-0-DELETE": "",
+        }
+        formset = InlineContributionFormset(data, instance=course, can_change_responsible=False, form_kwargs={'course': course})
+        self.assertTrue(formset.is_valid())
+
+        data["contributions-0-responsibility"] = "EDITOR"
+        formset = InlineContributionFormset(data, instance=course, can_change_responsible=False, form_kwargs={'course': course})
+        self.assertTrue(formset.is_valid())
+        self.assertTrue(formset.forms[0].fields["responsibility"].disabled)
```

Comment:
please assert that it's not disabled in the first run, and maybe drop a comment that for some reason, django does stuff differently in this case and you check for the disabledness therefore.